### DISCUSSION TITLE
Add support for Bitbucket Server tags as datasource

### DISCUSSION THREAD

**Discussion by ONordander** [renovatebot/renovate#24916] [Suggest an Idea]
### What would you like Renovate to be able to do?

Hi.
We would like to use Bitbucket Server tags as datasource with the Regex manager.

Renovate currently has support for [Bitbucket tags](https://github.com/renovatebot/renovate/tree/main/lib/modules/datasource/bitbucket-tags) however that's only compatible with Bitbucket Cloud.

### If you have any ideas on how this should be implemented, please tell us here.

I think the best approach is to add a new datasource for Bitbucket Server Tags.
It's also possible to add support for Bitbucket Server in Bitbucket Tags, however the APIs differ quite a bit so I think the cleaner approach is to create a separate datasource for Bitbucket Server.

Renovate already has support for Bitbucket Server as a platform so we should be able to re-use existing logic for authentication etc.

### Is this a feature you are interested in implementing yourself?

Yes

**Comment by PhilipAbed** [comment]
Hi @ONordander,
If you need any help don't hesitate to ask.
can i assign issue to you?

**Comment by ONordander** [comment]
> Hi @ONordander, If you need any help don't hesitate to ask. can i assign issue to you? 

Hey, thanks!
Sure you can assign it to me :)

**Comment by rarkins** [comment]
Consider also that git-refs should work with any platform

**Comment by PhilipAbed** [comment]
also make sure you take a look at https://github.com/renovatebot/renovate/blob/main/docs/development/best-practices.md

**Comment by Tilusch** [comment]
HI @ONordander have you made any progress on this or found a different solution?
